README.md

```diff
@@ -1,7 +1,26 @@
 Discard Old Build plugin
 ===========================
 
-Jenkins plugin to discard old builds with detailed configuration.
+Pantheon's fork of a Jenkins plugin to discard old builds with detailed configuration.
+
+We have added the option to discard all successful builds, beyond a certain number of kept builds. The existing plugin did not support this functionality out of the box.
+
+Typically we would want to configure our frequently-run jobs using this plugin option set to some numerical limit, e.g. 1000. The most recent 1000 builds will be kept, and then all successful builds beyond 1000 will be deleted. This will leave only the failed, aborted, etc. builds in the range of 1000+. Using it with Jenkin's default option `Discard Old Builds` with a higher limit, say 4000, and the net affect will be that we have unsuccessful builds saved for far longer.
```

Comment:
So in this example, Jenkins would keep 4000 builds.  The first 1000 would be a mix of success and failure, and the 1001-4000 would only be failures?